docs/README.md

```diff
@@ -1905,7 +1905,55 @@ See more details in [monitoring docs](#monitoring).
 
 See [this article](https://valyala.medium.com/how-victoriametrics-makes-instant-snapshots-for-multi-terabyte-time-series-data-e1f3fb0e0282) for more details.
 
-See also [how to work with snapshots](#how-to-work-with-snapshots).
+See also [how to work with snapshots](#how-to-work-with-snapshots) and [IndexDB](#indexdb).
+
+## IndexDB
+
+VictoriaMetrics identifies
+[time series](https://docs.victoriametrics.com/keyconcepts/#time-series) by
+`TSID` (time series ID) and stores
+[raw samples](https://docs.victoriametrics.com/keyconcepts/#raw-samples) sorted
+by TSID (see [Storage](#storage)). Thus, the TSID is a primary index and could
+be used for searching and retrieving raw samples. However, the TSID is never
+exposed to the clients, i.e. it is for internal use only.
+
+Instead, VictoriaMetrics maintains an **inverted index** that enables searching
+the raw samples by metric name, label name, and label value by mapping these
+values to the corresponding TSIDs.
+
+There are two types of inverted index: `global` and `per-day`. Initially,
+VictoriaMetrics had only global index. The per-day index has been added later as
+an optimization that speeds up data retrieval when
+[high churn rate](https://docs.victoriametrics.com/faq/#what-is-high-churn-rate)
+is the case.
+
+VictoriaMetrics decides which index to use based on the length of the search
+time range. If it is less than or equal to 40 days, then the per-day index is
+used. Otherwise, it will use the global index.
```

Comment:
I suggest using different formats instead of long text in this case, for your reference:

---
VictoriaMetrics uses two types of inverted indexes:
- Global Index: This was the original type of index used by VictoriaMetrics. It applies across the entire retention period, meaning that each MetricName -> TSID record is created only once per retention period.
- Per-day Index: This index was introduced later as an optimization for handling [high churn rates](https://docs.victoriametrics.com/faq/#what-is-high-churn-rate) (frequent creation and deletion of time series). The per-day index stores mappings similar to the global index but includes the date in each record, which speeds up data retrieval for queries within a shorter time range (up to 40 days).

When a search query is executed, VictoriaMetrics decides which index to use based on the time range of the query:
- Per-day Index is used if the search time range is 40 days or less.
- Global Index is used for search queries with a time range greater than 40 days.